1_basics.ipynb

```diff
@@ -143,23 +256,75 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "1.  Start the simulator environment by entering \"docker-compose up\" command.\n",
+    "\n",
+    "\n",
+    "2.  Access http://localhost:3000/ from your browser and open the simulator screen. The screen of Gazebo is displayed.\n",
+    "\n",
+    "![title](./imgs/1_gazebo.png)\n",
+    "\n",
+    "\n",
+    "3. Click the \"Start simulation\" button at the bottom left of the screen to start the simulation.\n",
+    "\n",
+    "![title](./imgs/1_play_button.png)\n",
+    "\n",
+    "\n",
+    "4. The simulation has started. Let's change the position and angle of the viewpoint by dragging the mouse.\n",
+    "   You can move the viewpoint in parallel by clicking the left button, rotate by clicking the wheel button, and zoom by scrolling the mouse wheel.\n",
```

Comment:
"wheel button" will likely not be understood.  But i t is common to say things like "click the mouse wheel" or "click the scroll wheel".

"4. The simulation has started. Let's change the position and angle of the viewpoint by dragging the mouse.\n",
 "   You can move the viewpoint in parallel by clicking the left button, rotate by clicking the **mouse wheel**, and zoom by scrolling the mouse wheel.\n",

Reply:
Thank you. "click the mouse wheel" is way nicer.